Svc/TlmPacketizer/docs/sdd.md

```diff
@@ -52,7 +52,7 @@ When a call to the `Run()` interface is called, the packet writes are locked and
 
 This diagram shows the scenario where telemetry channels are written to packets:
```

Comment:
And this.

Reply:
I'm assuming you're referring to dropping Section 3.3 then ? Since "3.3.1 External User Option" is the only subsection.

Reply:
Addressed in [c1cc7c9](https://github.com/nasa/fprime/pull/2651/commits/c1cc7c92cebcaa8d54bf5d33e073a7779953750f)